packages/docs-app/src/examples/select-examples/selectExample.tsx

```diff
@@ -82,16 +83,17 @@ export class SelectExample extends React.PureComponent<IExampleProps, ISelectExa
                     checked={this.state.resetOnSelect}
                     onChange={this.handleResetOnSelectChange}
                 />
-                <Switch
-                    label="Minimal popover style"
-                    checked={this.state.minimal}
-                    onChange={this.handleMinimalChange}
-                />
                 <Switch
                     label="Use initial content"
                     checked={this.state.hasInitialContent}
                     onChange={this.handleInitialContentChange}
                 />
+                <H5>Example</H5>
```

Comment:
this is more like `Popover props`: it's very much a supported concept, not something contrived for the example